### PR TITLE
Add linter for plaintext English

### PR DESCRIPTION
**Pull request by thomas-moulard**

### CODE REVIEW

**Comment by zmichaels11** [comment]
```
##[warning]Failed to download action 'https://api.github.com/repos/gaurav-nelson/github-action-vale-lint/tarball/v0.1.0'. Error Response status code does not indicate success: 404 (Not Found).
##[warning]Back off 10.777999999999999 seconds before retry.
##[warning]Failed to download action 'https://api.github.com/repos/gaurav-nelson/github-action-vale-lint/tarball/v0.1.0'. Error Response status code does not indicate success: 404 (Not Found).
##[warning]Back off 15.716 seconds before retry.
##[error]Response status code does not indicate success: 404 (Not Found).
```

**Comment by thomas-moulard** [comment]
The official action is currently broken, let's retry later.

**Comment by thomas-moulard** [comment]
Blocked on https://github.com/errata-ai/vale-action/issues/6

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -16,3 +16,20 @@ jobs:
...
+      uses: actions/checkout@master
+
+    - name: Vale
+      uses: errata-ai/vale-action@v1.0.3
+      with:
+        styles: |
+          https://github.com/errata-ai/write-good/releases/latest/download/write-good.zip
+          https://github.com/errata-ai/Google/releases/latest/download/Google.zip
+        config:
+          https://gist.githubusercontent.com/thomas-moulard/81e4fc260bddeb4149838b884ba86701/raw/cf331fe24d43fc2f02be4b40c5e83051a9e62130/vale.ini
```
Comment (piraka9011) [inline .github/workflows/lint.yml]: Do we want to host this as part of the repo or in a separate repo/gist?

**Pull request closed without merging**